CHANGELOG.next.asciidoc

```diff
@@ -26,6 +26,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Update to Golang 1.12.1. {pull}11330[11330]
 - Disable Alibaba Cloud and Tencent Cloud metadata providers by default. {pull}13812[12812]
 - API address is a required setting in `add_cloudfoundry_metadata`. {pull}21759[21759]
+- Cloud Foundry tags including metadata are moved to common metadata fields. {pull}22150[22150]
```

Comment:
Cloudfoundry tags will be added in 7.10.0. If we merge this change in time to include it in this release I will remove this changelog entry.

Reply:
Changelog entry removed, will re-add in a different PR if it doesn't reach 7.10.0.